openapi_tester/schema_tester.py

```diff
@@ -292,15 +297,17 @@ def _test_openapi_type_object(
                     response=data,
                     schema=schema_section,
                     reference=reference,
-                    hint="You need to add the missing schema key to the response, or remove it from the documented response.",
+                    hint="You need to add the missing schema key to the response, "
```

Comment:
I'd shorten this and all message, also remove any type of direct address to the user, i.e. "Add the missing key to the response or remove it from the schema."

Reply:
What do you think about changing this to `It is missing from the response or vice versa.`?

I was considering removing the hint altogether, but it does actually seem useful.

Reply:
i think its less clear. we should be as simple and straightforward as possible, i.e. this should be something you can brainlessly understand :)

Reply:
Do you have a suggestion for what we could use?

How about `The response should contain this key or the documentation should change`?